package.json

```diff
@@ -18,17 +28,29 @@
     "test"
   ],
   "dependencies": {
-    "async": "^3.1.0",
     "debug": "^4.1.1",
-    "lodash.defaults": "^4.0.0",
+    "JSONStream": "^1.3.5",
     "minimist": "^1.2.0",
     "mongodb-version-list": "^1.0.0",
```

Comment:
Looks like we now do this in this package - do we still need this dep?

Reply:
Ah yes, thank you  –we can drop this :)